addons/website/static/src/js/theme_preview_form.js

```diff
@@ -28,53 +27,49 @@ var ThemePreviewController = FormController.extend({
     /**
      * @override
      */
-    autofocus: function () {
-        // force refresh label of button switch
-        this.$switchButton = this._renderSwitchButton();
-        $('.o_switch_mode_button').replaceWith(this.$switchButton);
-        this._super.apply(this, arguments);
+    renderPager: function ($node, options) {
+        return this._super($node, options).then(() => {
+            this.$switcherButton = $(qweb.render('website.ThemePreview.SwitchModeButton'));
+            this.$switcherButton.appendTo(this.pager.$el);
```

Comment:
Why not doing this in `_updatePager` ?

Reply:
renderPager call the once, whoch add the button
_updatePager called several times that replacing existing button.

But we can do it in _updatePager with one if... if you prefer...